types/react-big-calendar/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for react-big-calendar 0.17.0
+// Type definitions for react-big-calendar 0.18.0
```

Comment:
Can you take out the disabled lint rules from `tslint.json` and fix up the version number here? It should be `0.18` instead of `0.18.0`

Reply:
@DanielRosenwasser: Fixed this, but I'm just wondering - shouldn't we have the exact same version as on [NPM](https://www.npmjs.com/package/react-big-calendar)? What's the reason for this?

Reply:
The reason is that the `@types` scope iterates on the patch version. Since the patch tag isn't really useful (it works alphabetically), we've resigned to major.minor, which works well enough for the most part.